requirements/base.txt

```diff
@@ -1,4 +1,4 @@
-Django==1.8.6
+Django==1.8.7
```

Comment:
Yeah, but it breaks things!



—
Patrick Forringer

On Tue, Dec 1, 2015 at 9:35 PM, Buddy Lindsey, Jr.
<notifications@github.com> wrote:

>> @@ -1,4 +1,4 @@
>> -Django==1.8.6
>> +Django==1.8.7
> Django 1.9 just released ;)
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/tulsawebdevs/tulsawebdevs.org/pull/26/files#r46374066